pkg/garden/app/docket.hoon

```diff
@@ -235,7 +271,35 @@
       ?~  p.sign  `state
       ((slog leaf+"Failed to uninstall %{(trip desk)}" u.p.sign) `state)
     ::
-        [%glob ~]
+        [%glob-ames ~]
+      ?-  -.sign
+        %kick      `state
+        %poke-ack  ~&([dap.bowl %unexpected-poke-ack] `state)
+      ::
+          %watch-ack
+        ?~  p.sign  `state
+        %-  %-  slog
+            leaf+"docket: failed to fetch glob over ames for {(trip desk)}"
+        ::TODO  maybe we want to retry, but with delay?
```

Comment:
I think this should cancel the installation.  Docket should display a notification that the app failed to install, and stop trying to install it.  The user could ask around and try again or something, but the system shouldn't keep trying indefinitely if something is broken on the remote.